extensions/eclipse-che-theia-plugin-remote/src/node/plugin-remote-init.ts

```diff
@@ -104,9 +118,22 @@ export class PluginRemoteInit {
 
         // bind local stuff
         inversifyContainer.load(pluginRemoteBackendModule);
-
+        inversifyContainer.load(new ContainerModule(bind => {
+            bind(CliManager).toSelf().inSingletonScope();
```

Comment:
AFAIK we don't need cli in remote

Reply:
Yes we do to be able to configure log levels at startup.